### PR TITLE
Fix x86 netci changes to build correctly.

### PR DESCRIPTION
**Pull request by jashook**
Also more verbosely show which jit and arch the job is.

### CODE REVIEW

**Comment by jashook** [comment]
@RussKeldorph @adiaaida @BruceForstall @mmitche ptal

**Comment by jashook** [comment]
@dotnet-bot test ci please.

**Comment by jashook** [comment]
Also if someone has a good eye for regex expressions please take a look

**Comment by jashook** [comment]
@dotnet-bot  test ci please

**Comment by jashook** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -214,6 +214,7 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
         suffix += '_bld'
     }
     def baseName = ''
+    def jit = 'ryujit'
```
Comment (adiaaida) [inline netci.groovy]: If this is only used in the x86 cases, and it is reset in the lb case, do we really need it to be defined here? Do we use it somewhere else?

File: netci.groovy [hunk 2]
```diff
@@ -1279,18 +1299,26 @@ combinedScenarios.each { scenario ->
                                 case 'x64':
                                 case 'x86ryujit':
                                 case 'x86lb':
+                                    def arch = ''
+                                    if (architecture == 'x64') {
+                                        arch = architecture
+                                    }
+                                    else {
+                                        arch = 'x86'
+                                    }
```
Comment (RussKeldorph) [inline netci.groovy]: Can we just disallow x86 on non-win for now?  We can't make sure these work, so I think it's best not to even try.  We probably won't support anything but RyuJIT on Unix anyway.

File: netci.groovy [hunk 3]
```diff
@@ -1095,10 +1108,17 @@ combinedScenarios.each { scenario ->
...
```
Comment (RussKeldorph) [inline netci.groovy]: How about:
def arch = architecture
if (architecture == 'x86ryujit' || architecture == 'x86lb) {
   arch = 'x86'
}

That way, it's correct as other architectures are added.

File: netci.groovy [hunk 4]
```diff
@@ -230,10 +231,13 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
             baseName = architecture.toLowerCase() + '_cross_' + configuration.toLowerCase() + '_' + os.toLowerCase()
             break
         case 'x86ryujit':
-            baseName = architecture.toLowerCase() + configuration.toLowerCase() + '_' + os.toLowerCase()
+            def arch = 'x86'
```
Comment (RussKeldorph) [inline netci.groovy]: I agree with adiaaia.  Why so many variables?  Seems like you might as well hardcode arch and jit inside the strings below.

**Comment by RussKeldorph** [comment]
Could you also update Documentation/project-docs/ci-trigger-phrases.md with this change?

**Comment by RussKeldorph** [comment]
:-1:

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Comment by jashook** [comment]
Updated to address suggestions

**Comment by jashook** [comment]
Updated documentation ptal. @dotnet-bot test ci please

**Comment by adiaaida** [comment]
LGTM

**Comment by RussKeldorph** [comment]
:+1:

**Pull request merged**